snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -74,9 +74,15 @@ def snappi_config(self):
         return self._config
 
     def get_config_object(self, name):
+        if self._config_objects.get(name):
+            raise NameError("{0} is not avialable any Device object"
+                            .format(name))
         return self._config_objects[name]
 
```

Comment:
The condition should be checking for None. But either way, I would propose a different approach.
It's more pythonic to use try/catch here (to avoid accessing dict twice in positive scenario):
```python
try:
    return dict[key]
except KeyError:
    raise NameError(msg)
```

Reply:
change it accordingly